### ISSUE TITLE
Drafts multiply during editing

### ISSUE THREAD

**Issue by gtandersen** [runbox/runbox7#1315]
**Describe the bug**
In some (large?) accounts, drafts that are being edited multiply during editing, resulting in 2-4 versions of the same draft in various states of completion.

The issue could be connected with drafts being open in Runbox 6, which also auto-saves drafts frequently.

**To Reproduce**
1. Go to https://runbox.com/app
2. Click to Compose (or reply/forward)
3. Start entering details 
4. Observe that the several versions of the same draft appear

**Expected behavior**
Only one version of the same draft should be created.

Perhaps there are ways to simplify Drafts functionality further so that only the draft you have open is saved/refreshed?

**Screenshots**

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Firefox

**Comment by kotecky** [comment]
Same behaviour with latest Firefox on Win10.

**Comment by piratefinn** [comment]
I've attempted to reproduce this bug in multiple ways on both Firefox and Chrome on Win10 along with IMAP via Thunderbird, and am failing to. Tried a variety of scenarios such as creating a fresh mail, replying to other mails and the like, having it open in both chrome and firefox at once, having the email client also open, typing in one browser then refreshing on the other, and all sorts. Therefore so far, I think more information is needed on reproducing the bug as currently, am struggling.
